packages/settings-library/src/settings_library/prometheus.py

```diff
@@ -1,26 +1,35 @@
 from functools import cached_property
 
-from pydantic.networks import AnyHttpUrl
+from pydantic import AnyUrl, SecretStr
 from settings_library.base import BaseCustomSettings
 from settings_library.utils_service import MixinServiceSettings
 
-from .basic_types import PortInt, VersionTag
+from .basic_types import VersionTag
 
 
 class PrometheusSettings(BaseCustomSettings, MixinServiceSettings):
-    PROMETHEUS_HOST: str = "prometheus"
-    PROMETHEUS_PORT: PortInt = 9090
-    PROMETHEUS_VTAG: VersionTag = "v1"
+    PROMETHEUS_URL: AnyUrl
```

Comment:
Q: Normally any service base url are captured using `*_HOST` and `*_PORT`.  If i recall well, we started with `URL` but then decided to split it in these two parts. Any particular reason why to go back to `URL`?

Reply:
HOST and PORT do not allow to change the scheme in case of need.